topics/assembly/tutorials/unicycler-assembly/slides.html

```diff
@@ -0,0 +1,1143 @@
+---
+layout: introduction_slides
+topic_name: unicycler_assembly
+tutorial_name: am_genome_assembly
```

Comment:
tutorial name should match the name in the metadata file, so `unicycler-assembly`